### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by jayrangnani-gl**
### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. ...
2. ...

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by jayrangnani-gl** [comment]
@magento run all tests

**Comment by jayrangnani-gl** [comment]
@magento run all tests

**Comment by jayrangnani-gl** [comment]
@magento run all tests

**Comment by jayrangnani-gl** [comment]
@magento run all tests

**Comment by jayrangnani-gl** [comment]
@magento run all tests

**Comment by jayrangnani-gl** [comment]
@magento run all tests

**Comment by jayrangnani-gl** [comment]
@magento run all tests

**Comment by jayrangnani-gl** [comment]
@magento run all tests

**Comment by jayrangnani-gl** [comment]
@magento run all tests

File: dev/tests/static/framework/Magento/TestFramework/Inspection/WordsFinder.php [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
+ *
+ *  NOTICE: All information contained herein is, and remains
+ *  the property of Adobe and its suppliers, if any. The intellectual
+ *  and technical concepts contained herein are proprietary to Adobe
+ *  and its suppliers and are protected by all applicable intellectual
+ *  property laws, including trade secret and copyright laws.
+ *  Dissemination of this information or reproduction of this material
+ *  is strictly forbidden unless prior written permission is obtained
+ *  from Adobe.
+ *  ************************************************************************
```
Comment (fredden) [inline dev/tests/static/framework/Magento/TestFramework/Inspection/WordsFinder.php]: This license change is unexpected. Please can you explain why this has happened?

**Pull request closed without merging**